app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= full_title (yield (:title)) %></title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
+
+    <%= render "layouts/header" %>
+
+    <div class="container">
+      <%= yield %>
+    </div>
+
   </body>
-</html>
+</html>
```

Comment:
fix endline please

Reply:
at line 16, do i need to delete the line? is it correct i do this?